src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random RANDOM = new Random();
+
+    public Color getRandomColor() {
+        int index = RANDOM.nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
The 'getRandomColor' method does not have an access modifier. It's a good practice to define access modifiers explicitly.